src/test/java/seedu/sellsavvy/logic/commands/ordercommands/EditOrderCommandTest.java

```diff
@@ -0,0 +1,169 @@
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import seedu.sellsavvy.commons.core.index.Index;
+import seedu.sellsavvy.logic.Messages;
+import seedu.sellsavvy.logic.commands.generalcommands.ClearCommand;
+import seedu.sellsavvy.logic.commands.ordercommands.EditOrderCommand.EditOrderDescriptor;
+import seedu.sellsavvy.model.Model;
+import seedu.sellsavvy.model.ModelManager;
+import seedu.sellsavvy.model.UserPrefs;
+import seedu.sellsavvy.model.order.Order;
+import seedu.sellsavvy.model.person.Person;
+import seedu.sellsavvy.testutil.EditOrderDescriptorBuilder;
+import seedu.sellsavvy.testutil.OrderBuilder;
+
+public class EditOrderCommandTest {
+    private Model model;
+    private Person selectedPerson;
+
+    @BeforeEach
+    public void setUp() {
+        model = new ModelManager(getTypicalAddressBook(), new UserPrefs()).createCopy();
+        selectedPerson = model.getFilteredPersonList().get(INDEX_FOURTH_PERSON.getZeroBased());
+        model.updateSelectedPerson(selectedPerson);
+    }
+
+    @Test
+    public void execute_allFieldsSpecifiedUnfilteredList_success() {
+        Order editedOrder = new OrderBuilder().build();
+        EditOrderDescriptor descriptor = new EditOrderDescriptorBuilder(editedOrder).build();
+        EditOrderCommand editOrderCommand = new EditOrderCommand(INDEX_FIRST_ORDER, descriptor);
+
+        String expectedMessage = String.format(EditOrderCommand.MESSAGE_EDIT_ORDER_SUCCESS,
+                Messages.format(editedOrder));
+
+        Model expectedModel = model.createCopy();
+        expectedModel.setOrder(expectedModel.getFilteredOrderList().get(INDEX_FIRST_ORDER.getZeroBased()), editedOrder);
+
+        assertCommandSuccess(editOrderCommand, model, expectedMessage, expectedModel);
+    }
+
+    @Test
+    public void execute_someFieldsSpecifiedUnfilteredList_success() {
+        Index indexLastOrder = Index.fromOneBased(model.getFilteredOrderList().size());
+        Order lastOrder = model.getFilteredOrderList().get(indexLastOrder.getZeroBased());
+
+        OrderBuilder orderInList = new OrderBuilder(lastOrder);
+        Order editedOrder = orderInList.withCount(VALID_COUNT_ATLAS)
+                .withDate(VALID_DATE_ATLAS).build();
+
+        EditOrderDescriptor descriptor = new EditOrderDescriptorBuilder().withQuantity(VALID_COUNT_ATLAS)
+                .withDate(VALID_DATE_ATLAS).build();
+        EditOrderCommand editOrderCommand = new EditOrderCommand(indexLastOrder, descriptor);
+
+        String expectedMessage = String.format(EditOrderCommand.MESSAGE_EDIT_ORDER_SUCCESS,
+                Messages.format(editedOrder));
+
+        Model expectedModel = model.createCopy();
+        expectedModel.setOrder(expectedModel.getFilteredOrderList().get(indexLastOrder.getZeroBased()), editedOrder);
+
+        assertCommandSuccess(editOrderCommand, model, expectedMessage, expectedModel);
+    }
+
+    @Test
+    public void execute_onOrderListInDisplay_failure() {
```

Comment:
```suggestion
    public void execute_noOrderListDisplayed_failure() {
```
Mispelled and can this be rephrased to standardize with MarkOrderCommandTest and DeleteOrderCommandTest?

Reply:
fixed